ui/src/assets/i18n/de.json

```diff
@@ -273,5 +273,10 @@
     "TOPIC_PRODUCER_APPLICATION_ADDED_STAGING": "Produzent <code>{{producer}}</code> hinzufügen für Topic <code>{{topicName}}</code>",
     "TOPIC_PRODUCER_APPLICATION_REMOVED_STAGING": "Produzent <code>${this.applicationInfo(change.producerApplicationId).name} ` + `</code> entfernen für Topic <code>{{topicName}}</code>",
     "TOPIC_SUBSCRIPTION_APPROVAL_REQUIRED_FLAG_UPDATED_STAGING": "Für Topic <code>{{topicName}}</code> die Freigabe von Abonnements erforderlich machen",
-    "OTHER_CHANGE_STAGING": "Änderung vom Typ <code>{{changeType}}</code> durchführen"
+    "OTHER_CHANGE_STAGING": "Änderung vom Typ <code>{{changeType}}</code> durchführen",
+    "Events": "Ereignisse",
+    "Master Data": "Stammdaten",
+    "Commands": "Befehle",
+    "App internal": "Interne App"
```

Comment:
I'd prefer using the english words here, as we never translate at least the first three. As suggested above, if enum keys are used, this would be `"EVENTS": "Events"`, and also in the `en.json`. 

"INTERNAL" could be mapped to "Anwendungsintern" in german, that would be fine.